extension/popup/popup.jsx

```diff
@@ -0,0 +1,744 @@
+/* globals util, voice, vad, lottie, settings, log, voiceShim, buildSettings */
+
+const React = window.React
```

Comment:
If you put `React` and `ReactDOM` in the `/* globals */` section then these assignments won't be necessary.